### PR TITLE
SONARCSANA-113 Any() should be used to test for emptiness

### PR DESCRIPTION
**Pull request by tvsonar**

### CODE REVIEW

File: src/Tests/CSharp.CodeAnalysis.UnitTest/Verifier.cs [hunk 1]
```diff
@@ -80,6 +80,7 @@ public static void Verify(string path, DiagnosticAnalyzer diagnosticAnalyzer)
             {
                 var document = workspace.CurrentSolution.AddProject("foo", "foo.dll", LanguageNames.CSharp)
                     .AddMetadataReference(MetadataReference.CreateFromAssembly(typeof(object).Assembly))
+                    .AddMetadataReference(MetadataReference.CreateFromAssembly(typeof(Enumerable).Assembly))
```
Comment (dbolkensteyn) [inline src/Tests/CSharp.CodeAnalysis.UnitTest/Verifier.cs]: @tvsonar Could you briefly explain why this change is required? Will this rule work with the current rule runner?

Comment (tvsonar) [inline src/Tests/CSharp.CodeAnalysis.UnitTest/Verifier.cs]: @dbolkensteyn It was needed because the semantic analyzer can only find the extension methods if it has a reference to the assembly.
The rule runner will not fail. But we should probably add this reference to the ```CompilationHelper``` as well.

**Comment by dbolkensteyn** [comment]
@tvsonar LGTM - I let you decide if any change should be made to the rule runner
